### PR TITLE
(7.1.x) LRDOCS-8195 ES7 

### PR DESCRIPTION
**Pull request by lipusz**
https://issues.liferay.com/browse/LRDOCS-8195

Hey Russ,

I've made some changes that I think make sense.

Regards,
Tibor

### CODE REVIEW

**Comment by lipusz** [comment]
Also, the PEM-format setup was missing `ssl.key_passphrase` for the private key.

**Comment by lipusz** [comment]
The very same config works fine with both 6.5.x+ and 7.x so I decided to save some space and simplified the content.

**Comment by lipusz** [comment]
Also edited the header names to better reflect what they are covering and also to match more-or-less with Elastic's docs/terminology.

**Comment by rbohl** [comment]
Merged Locally.

File: en/discover/deployment/articles-dxp/02-installing-search-engine/02-enterprise-search/01-security/00-security-intro.markdown [hunk 1]
```diff
@@ -171,3 +287,61 @@ Here's the complete list of security configuration options:
...
+xpack.security.transport.ssl.verification_mode: certificate
+
+# TLS/SSL settings for HTTP layer (PKCS#12)
+xpack.security.http.ssl.enabled: true
+xpack.security.http.ssl.keystore.path: certs/elastic-nodes.p12
+xpack.security.http.ssl.keystore.password: liferay
+xpack.security.http.ssl.truststore.path: certs/elastic-nodes.p12
+xpack.security.http.ssl.truststore.password: liferay
+
+# Comment out when Kibana and Liferay's LES Monitoring are also configured
```
Comment (rbohl) [inline en/discover/deployment/articles-dxp/02-installing-search-engine/02-enterprise-search/01-security/00-security-intro.markdown:318]: Do you mean to say _uncomment_ (remove the leading `#`) or did you really mean _Comment out_ (add the leading `#`)?

Comment (rbohl) [reply en/discover/deployment/articles-dxp/02-installing-search-engine/02-enterprise-search/01-security/00-security-intro.markdown:318]: I propose to change it to 

```properties
# Use this setting when Kibana and Liferay's LES Monitoring are also configured
#xpack.monitoring.collection.enabled: true
```

**Comment by rbohl** [comment]
Superseded: https://github.com/sez11a/liferay-docs/pull/4949

**Comment by lipusz** [comment]
Ah, yes

Sent from mobile
--
Product Supportability Manager & SME for Search & Security

**Comment by lipusz** [comment]
Could we send to JR instead?

Sent from mobile
--
Product Supportability Manager & SME for Search & Security

**Comment by rbohl** [comment]
> Could we send to JR instead?

No, he doesn't look at `liferay-docs`

**Pull request closed without merging**